src/DacpacTool/Program.cs

```diff
@@ -132,6 +142,33 @@ private static int BuildDacpac(BuildOptions options)
             return 0;
         }
 
+        private static int InspectIncludes(InspectOptions options)
+        {
+            // Wait for a debugger to attach
+            WaitForDebuggerToAttach(options.Debug);
+
+            var packageInspector = new ScriptInspector();
```

Comment:
Might want to rename the variable here as well.

Reply:
oops 🤦

Reply:
Fixed.